pynestml/codegeneration/nest_builder.py

```diff
@@ -141,21 +161,25 @@ def build(self) -> None:
 
         # first call cmake with all the arguments
         try:
-            subprocess.check_call(cmake_cmd, stderr=subprocess.STDOUT, shell=shell,
-                                  cwd=str(os.path.join(target_path)))
+            result = subprocess.check_call(cmake_cmd, stderr=error_file, shell=shell, stdout=output_file,
```

Comment:
``result`` is anyway ignored, right?
```suggestion
            subprocess.check_call(cmake_cmd, stderr=error_file, shell=shell, stdout=output_file,
```